### ISSUE TITLE
Matplotlib fails for mpl

### ISSUE THREAD

**Issue by tigerjack** [Qiskit/qiskit-terra#2010]
### Information

- **Qiskit Terra version**: latest from master
- **Python version**: 3.6
- **Operating system**: Linux 64bit
- **matplotlib version**:  3.0.2 and 3.0.3

### What is the current behavior?
Running `circuit_drawer(qc, filename='a.png', output='mpl')` with whatever circuit returns
```
Traceback (most recent call last):
  File "/mnt/residential_zone/home/bla/qiskit/quantum-isd/test/test_lee_tmp.py", line 81, in common                            
    output='mpl')
  File "/mnt/residential_zone/home/bla/virtual_envs/qiskit_beta/src/qiskit-terra/qiskit/tools/visualization/_circuit_visualization.py", line 195, in circuit_drawer
    justify=justify)
  File "/mnt/residential_zone/home/bla/virtual_envs/qiskit_beta/src/qiskit-terra/qiskit/tools/visualization/_circuit_visualization.py", line 462, in _matplotlib_circuit_drawer
    return qcd.draw(filename)
  File "/mnt/residential_zone/home/bla/virtual_envs/qiskit_beta/src/qiskit-terra/qiskit/tools/visualization/_matplotlib.py", line 335, in draw
    self._draw_ops(verbose)
  File "/mnt/residential_zone/home/bla/virtual_envs/qiskit_beta/src/qiskit-terra/qiskit/tools/visualization/_matplotlib.py", line 485, in _draw_ops
    if op['name'] in _wide_gate:
TypeError: 'DAGNode' object is not subscriptable

```

**Comment by maddy-tod** [comment]
This is fixed by #2001. @kdk or @1ucian0 could you please take a look and get that PR merged?

**Comment by 1ucian0** [comment]
Closed via #2001.
